src/Orders/OrderModel.php

```diff
@@ -36,6 +29,9 @@ class OrderModel extends Model
         'gateway' => 'json',
         'data' => 'json',
         'paid_date' => 'datetime',
+
+        // Ignore this. Needed for SC's test suite.
+        'ordered_on_tuesday' => 'boolean',
```

Comment:
We can probably change this `ordered_on_tuesday` to not have a casted value, then we won't need it here.